chore(app-store): Update generated apps list

tennisleng opened the pull request:
Splitting #25660: 


No video is needed for this PR. The changes are purely performance-related:

What it changes:
- How the app list data is loaded in the Kbar (command bar / ⌘K menu)
- Changes from static import to lazy/dynamic import



<!-- This is an auto-generated description by cubic. -->
---
## Summary by cubic
Generates a lightweight apps.list.generated.json and lazy-loads it in Kbar to reduce bundle size and speed up the command bar. Also adds a build step and tweaks Next.js config for better kbar import optimization.

- **Refactors**
  - app-store-cli: generate apps.list.generated.json with { name, slug } for each app.
  - Kbar: dynamically import the list and build actions at runtime; remove static appStoreMetadata import.
  - next.config.js: add "kbar" to optimizePackageImports.

<sup>Written for commit 96970a405ea2aaf9ee7f30142c0b6982d0c1dc04. Summary will update automatically on new commits.</sup>

<!-- End of auto-generated description by cubic. -->

tennisleng committed:
perf(kbar): Lazy load app store data logic

tennisleng committed:
chore(app-store): Update generated apps list

volnei reviewed (changes requested):
hey @tennisleng thank you for your contribution ❤️
Can you check cubic-dev comments, please?

tennisleng committed:
fix: address cubic dev comments

- Fix app name generation to prefer 'title' from metadata over directory name
- Remove developer-specific script split_calcom_25660.sh
- Regenerate apps.list.generated.json with correct 'Wipe my cal' name

Pallava-Joshi commented on packages/app-store-cli/src/build.ts:
```diff
@@ -456,6 +456,32 @@ function generateFiles() {
+
+    if (fs.existsSync(configPath)) {
+      const rawConfig = fs.readFileSync(configPath, "utf8");
+      const parsedConfig = JSON.parse(rawConfig);
```
JSON.parse(rawConfig) can throw error if config.json is malformed, please wrap around try-catch

pallava-joshi reviewed (changes requested):
please implement devin suggestions and resolve merge conflicts.
